extensions/big-data-cluster/src/bigDataCluster/controller/clusterControllerApi.ts

```diff
@@ -275,9 +275,9 @@ export class ClusterController {
 			mountPath);
 	}
 
-	private async deleteMountImpl(mountPath: string): Promise<MountResponse> {
-		let auth = await this._authPromise;
-		const api = new DefaultApiWrapper(this._username, this._password, this._url, auth);
+	private async deleteMountImpl(self: ClusterController, mountPath: string): Promise<MountResponse> {
```

Comment:
to avoid this you can also do ()=>{ this.deleteMountImpl(...)} when you call it.